tests/functional/object/mcg/test_noobaa_metrics.py

```
@@ -218,3 +218,175 @@ def test_noobaa_space_available_using_cli(
+            use_yes=True,
+        )
+        quota_status = get_bucket_status_value(mcg_obj, bucket_name, "QuotaStatus")
+        assert (
+            quota_status == QuotaStatus.OPTIMAL
+        ), f"Quota status after setting max-size is {quota_status}, expected {QuotaStatus.OPTIMAL}"
+        logger.info(f"Max-size quota set to {max_size_gi}Gi, status is OPTIMAL")
+
+        # 3. Write ~88% of quota (9 x 200MB = 1.8GB, ~88% of 2Gi)
+        #    This exceeds the 80% approaching threshold
+        logger.info(
+            f"Writing {approaching_count} objects of {obj_size_mb} to approach quota"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)